ddtrace/tracer/log.go

```diff
@@ -72,6 +73,10 @@ func checkEndpoint(endpoint string) error {
 // logStartup generates a startupInfo for a tracer and writes it to the log in
 // JSON format.
 func logStartup(t *tracer) {
+	serviceMappings := make(map[string]string)
+	for svc, newSvc := range t.config.serviceMappings {
+		serviceMappings[svc] = fmt.Sprintf("%v", newSvc)
+	}
```

Comment:
Can you de-duplicate this code and make a `flattenMap` function?
```go
func flattenMap(m map[string]string) string {
    // ...
}
```
Then we can use that for both `tags` and `serviceMappings`

Reply:
Ended up passing `t.config.serviceMappings` directly in the info struct so the code is not duplicated anymore.

Reply:
Perfect!